### PR TITLE
adds investments to Plaid Node

### PR DESCRIPTION
**Pull request by rayraycano**
This PR adds two new routes to the Plaid Node client
* /investments/transactions/get
* /investments/holdings/get

### CODE REVIEW

**Comment by rayraycano** [comment]
cc @michaelckelly

**Comment by rayraycano** [comment]
Retest this please.

File: README.md [hunk 1]
```diff
@@ -111,6 +111,12 @@ plaidClient.getIncome(access_token, cb);
 // getCreditDetails(String, Function)
 plaidClient.getCreditDetails(access_token, cb);
 
+// getHoldings(String, Function)
+plaidClient.getHoldings(access_token, cb);
+// getInvestmentTransactions(String, Date(YYYY-MM-DD), Date(YYYY-MM-DD),
+// Object?, Function)
+plaidClient.getInvestmentTransactions(access_token, cb);
```
Comment (pbernasconi) [inline README.md]: Are params missing here. Looking at the equivalent python function it seems so:
https://github.com/plaid/plaid-python/pull/169/files#diff-469bbc1fb03163938e0174980a8059c5R7

File: index.d.ts [hunk 2]
```diff
@@ -188,11 +211,28 @@ declare module 'plaid' {
 
   interface Holding {
     account_id: string;
-    symbol: string | null;
+    security_id: string;
+
+    institution_value: number | null;
+    institution_price: number | null;
+    quantity: number | null;
+    institution_price_as_of: string  | null;
```
Comment (pbernasconi) [inline index.d.ts]: nit: extra space after `string   `

File: index.d.ts [hunk 3]
```diff
@@ -188,11 +211,28 @@ declare module 'plaid' {
 
   interface Holding {
     account_id: string;
-    symbol: string | null;
+    security_id: string;
+
```
Comment (pbernasconi) [inline index.d.ts]: nit: line-break

File: index.d.ts [hunk 4]
```diff
@@ -600,7 +648,12 @@ declare module 'plaid' {
     getCreditDetails: AccessTokenFn<CreditDetailsResponse>;
     // getHoldings(String, Function)
     getHoldings: AccessTokenFn<HoldingsResponse>;
-
+    // getInvestmentTransactions(String, Date, Date, Function)
+    getInvestmentTransactions(accessToken: string,
+                    startDate: Iso8601DateString,
```
Comment (pbernasconi) [inline index.d.ts]: nit: 📐

File: README.md (hunk 1 above)
Comment (rayraycano) [reply README.md]: good call out! updated

File: index.d.ts (hunk 2 above)
Comment (rayraycano) [reply index.d.ts]: ⚡️

File: index.d.ts (hunk 3 above)
Comment (rayraycano) [reply index.d.ts]: ⚡️

File: index.d.ts (hunk 4 above)
Comment (rayraycano) [reply index.d.ts]: :zap:

File: package-lock.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "plaid",
-  "version": "3.1.1",
+  "version": "4.0.0",
```
Comment (pbernasconi) [inline package-lock.json]: How about we make this a minor bump to 3.2.2? There don't seem to be any breaking changes, so I don't see a need to major bump it

Comment (rayraycano) [reply package-lock.json]: Interesting -- I don't think I updated this myself. Does it update as part of `make setup`? regardless, going to 3.2.0

Comment (rayraycano) [reply package-lock.json]: :zap

File: package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "plaid",
-  "version": "4.0.0",
+  "version": "3.2.0",
```
Comment (pbernasconi) [inline package.json]: my mistake before – looks like @michaelckelly released 4.0 15 days ago – so this would be `4.1.0`

**Pull request merged**